tasks/issue.py

```diff
@@ -40,3 +40,59 @@ def assign_owner(_, issue_id, dry_run=False):
 @task
 def generate_model(_):
     fetch_data_and_train_model()
+
+
+WAVES = [
+    "wave",
+    "waveboi",
+    "wastelands-wave",
+    "wave_hello",
+    "wave-hokusai",
+    "wave_moomin",
+    "wave2",
+    "wave3",
+    "wallee-wave",
+    "vaporeon_wave",
+    "turtle-wave",
+    "softwave",
+    "shiba-wave",
+    "minion-wave",
+    "meow_wave_comfy",
+    "mario-wave",
+    "link-wave",
+    "kirby_wave",
+    "frog-wave",
+    "fox_wave",
+    "duckwave",
+    "cyr-wave",
+    "cozy-wave",
+    "cat-wave",
+    "capy-wave",
+    "bufo-wave",
+    "bongo-wave",
+    "blobwave",
+    "birb-wave",
+    "arnaud-wave",
+]
+
+
+@task
+def ask_reviews(_, pr_id):
+    gh = GithubAPI()
+    pr = gh.repo.get_pull(int(pr_id))
+    if any(label.name == 'ask-review' for label in pr.get_labels()):
+        reviewers = [f"@datadog/{team.slug}" for team in pr.requested_teams]
+        import random
+
+        from slack_sdk import WebClient
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you move this to the function top / the module top?

Reply:
Done